test/system/maintenance_tasks/runs_test.rb

```diff
@@ -29,6 +29,34 @@ class RunsTest < ApplicationSystemTestCase
       assert_no_button "Run"
     end
 
+    test "run a Task that accepts parameters" do
+      visit maintenance_tasks_path
+
+      click_on("Maintenance::ParamsTask")
+      post_id = Post.first.id
+      fill_in("[task_arguments][post_ids]", with: post_id.to_s)
```

Comment:
Is that the `name` of the field? It's kind of weird. Though it's probably ok since it's not an API and just a browser field. I wonder if we could change the test to use the label name instead of the field name though… 🤔

Reply:
Yep, we can change this to use the label's `for` (the textarea's `id` instead of `name`)